packages/dashboard-v2/README.md

```diff
@@ -0,0 +1,10 @@
+# Skynet Account Dashboard
+
+Code behind [account.skynetpro.net](https://account.skynetpro.net/)
+
+
+## Development
+This is a Gatsby application. To run it locally, all you need is:
+
+* `npm install`
+* `npm run start`
```

Comment:
we're using yarn everywhere else, I would appreciate consistency unless there is a case why we should use npm instead

Reply:
Cool 👍  00eb919